<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ballistics</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="wind">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;0 м/с&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 45px;
color: white;
background: none;
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>ВЫХОД</string>
   </property>
  </widget>
  <widget class="QLabel" name="carriage_first">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="files.qrc">:/images/carriage_1_small.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="carriage_second">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>500</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="files.qrc">:/images/carriage_2_small.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="cannon_first" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="cannon_second" native="true">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>500</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fire_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>550</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 45px;
color: white;
background: none;
background-color: rgb(220, 25, 0);</string>
   </property>
   <property name="text">
    <string>Огонь!</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_left">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>660</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>45</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt; line-height: 16px;&quot;&gt;Угол&lt;br&gt;Масса&lt;br&gt;Мощь&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>0</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>0°</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_angle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
</string>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mass">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>20</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>1 кг</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_power">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="power">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>40</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>0 ед.</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_mass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_right">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>660</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_menu_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>45</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt; line-height: 16px;&quot;&gt;Угол&lt;br&gt;Масса&lt;br&gt;Мощь&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>0</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>0°</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_angle_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
</string>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mass_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>20</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>1 кг</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_power_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="power_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>40</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>0 ед.</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_mass_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="Ball" name="left_ball">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>584</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Ball" name="right_ball">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>584</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="info" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 45px;
color: white;
background: none;
background-color: rgb(0, 85, 255);</string>
   </property>
   <widget class="QLabel" name="text_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>600</width>
      <height>275</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добро пожаловать в Ballistics, простую в освоении баллистическую игру!&lt;br/&gt;&lt;br/&gt;Правила просты: настрой массу ядра, а также силу и угол выстрела при помощи ползунков. Готов выстрелить - жми &amp;quot;Огонь!&amp;quot;. Дальше то же самое нужно сделать второму игроку.&lt;br/&gt;&lt;br/&gt;Победит тот, кто попадет ядром в пушку противника. &lt;br/&gt;Не забудь, что на полет снаряда влияет ветер.&lt;/p&gt;&lt;p&gt;Меткого глаза!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="close_info">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: white;
background: none;
background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Понятно, поехали!</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="win_widget" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>700</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 45px;
color: white;
background: none;
background-color: rgb(0, 85, 255);</string>
   </property>
   <widget class="QPushButton" name="win_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QLabel" name="win_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>600</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <zorder>right_ball</zorder>
  <zorder>left_ball</zorder>
  <zorder>endButton</zorder>
  <zorder>wind</zorder>
  <zorder>cannon_first</zorder>
  <zorder>carriage_first</zorder>
  <zorder>cannon_second</zorder>
  <zorder>carriage_second</zorder>
  <zorder>fire_button</zorder>
  <zorder>frame_left</zorder>
  <zorder>frame_right</zorder>
  <zorder>info</zorder>
  <zorder>win_widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ball</class>
   <extends>QLabel</extends>
   <header>ball.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
